pset-iterators.js

```diff
@@ -66,6 +71,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter(function(singularPresident) {
+  return singularPresident.president.split(" ")[0] === "James"
```

Comment:
Really great use of object traversing, method chaining and conditionals all in one line!